app/assets/javascripts/Components/Result/tags_panel.jsx

```diff
@@ -47,13 +47,11 @@ export class TagsPanel extends React.Component {
       <div>
         <h4>{I18n.t('tags.results.current_tags')}</h4>
         <ul className='tag-list'>{this.renderTagList()}</ul>
-        {!this.props.remark_submitted &&
-         (<div>
+         <div>
```

Comment:
I think you can actually delete this `div` altogether, as the reason it was used before was for grouping with the `&&` expression.

Then, make sure to unindent the following two lines so that the match the previous two lines.